Blog/view/sessions.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+session_start();
+
+if(!isset($_SESSION['userName'])){
```

Comment:
If this is just practice code, that's fine. If it's the real login system, I'd suggest storing the userID (i.e. the primary key of the user table). There are a variety of reasons for this, but the main one is that internally, the primary key is meant to be the "pointer" to a row in your db. The username is just the thing a user can remember easily, but it's not ideal to use as a pointer in your internal code.